src/rpc/util.cpp

```diff
@@ -150,8 +151,8 @@ CPubKey AddrToPubKey(CKeyStore* const keystore, const std::string& addr_in)
     return vchPubKey;
 }
 
-// Creates a multisig redeemscript from a given list of public keys and number required.
-CScript CreateMultisigRedeemscript(const int required, const std::vector<CPubKey>& pubkeys)
+// Creates a multisig address from a given list of public keys, number required, and the address type
+CTxDestination AddAndGetMultisigDestination(CKeyStore& keystore, CScript& script_out, const int required, const std::vector<CPubKey>& pubkeys, OutputType type)
```

Comment:
nit: as an out arg, it'd be somewhat more clear to place `script_out` towards the end. `AddAndGetDestinationForScript`'s placement is based on the same not being an out arg.

Reply:
yeah, that seems to en the convention, usually
wish C++ had tuple unpacking and it was unnecessary to bother with 'output arguments'

Reply:
Done